### ISSUE TITLE
Delete team

### ISSUE THREAD

**Issue by svitlanakalancha** [hpi-swt2/sport-portal#22]
As Team Owner I can delete my team anytime.

**Acceptance criteria:**

- [x] Only Team Owner or Admin have rights to delete the team.

- [x] Team can't be deleted, if it participates in any match or event.  

- [x] Show success or error message according to the case (possibly out of scope)

**Comment by bakoe** [comment]
depends on #27: *Roles in team*

**Comment by svitlanakalancha** [comment]
already exists. TO DO: set roles

**Comment by lheilmann** [comment]
Depends on #23

**Comment by lheilmann** [comment]
Friendly reminder: Please list the development tasks before implementing the feature. See #26 for example.

**Comment by conradlempert** [comment]
**Development Tasks**

When trying to delete:
- [x] Check if the deleting user is admin or team owner
- [ ] Check if the team participates in any match or event
- [ ] Show an error message if its not possible

- [ ] Test everything
